components/login/LoginWithGoogleButton.vue

```diff
@@ -19,7 +21,7 @@ async function loginWithGoogle() {
   }
   const googleSignInRequest = await userStore.loginWithGoogle(auth, props.login)
   if (googleSignInRequest.success) {
-    navigateTo(localePath("/overview"));
+    navigateTo(localePath(redirectTo ?? 'overview'));
```

Comment:
here it changed in the other direction

Reply:
That is ts code so the formater changed it the other way around